docs/core/getting-started/install/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Install .NET Core
+description: Install .NET Core
+keywords: .NET, .NET Core, C#, Getting Started, Acquisition, Cross Platform
+author: kendrahavens
+ms.author: kendrahavens
+ms.date: 12/19/2016
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 
+---
+# Install .NET Core
```

Comment:
Are we covering Installing the runtime or the SDK in the topics? I believe it's the latter, so should we call this topic Install the .NET Core SDK?

I'm also wondering if we really need this topic. For example, the Install node doesn't HAVE to have a topic associated. It makes sense if we think we want to link to this topic and not to the child topics directly and if we think it's better organized here if the OS groupings.